app/src/main/java/cm/aptoide/pt/view/app/AppService.java

```diff
@@ -249,13 +251,29 @@ public AppService(StoreCredentialsProvider storeCredentialsProvider,
               app.hasBilling(), app.hasAdvertising(), app.getBdsFlags(), app.getAge()
               .getRating() == MATURE_APP_RATING, app.getFile()
               .getSignature()
-              .getSha1());
+              .getSha1(), app.hasSplits() ? map(app.getAab()
+              .getSplits()) : Collections.emptyList(), app.hasSplits() ? app.getAab()
```

Comment:
In my opinion Aab pojo should reflect exactly what comes from the web, then we wrap it with other layers with that logic. I don't think it is worth the effort here. to make all the changes because of these.